fiware-region-sanity-tests/tests/fiware_region_base_tests.py

```diff
@@ -28,6 +28,9 @@
 from commons.constants import *
 from novaclient.exceptions import Forbidden, OverLimit
 from datetime import datetime
+from commons.ssh_client import SSHClient
+from paramiko.ssh_exception import AuthenticationException
```

Comment:
It is needed to import AuthenticationException because I am only importing SSHClient from *commons.ssh_client*, but I will import it from *ssh_client* module instead of *paramiko*.